website/blog/2021-09-29-release-apache-apisix-2.10-en.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Release Apache APISIX 2.10.0"
+author: "Zexuan Luo"
+authorURL: "https://github.com/spacewander"
+authorImageURL: "https://avatars.githubusercontent.com/u/4161644?v=4"
+keywords:
+- APISIX
+- Apache APISIX
+- Release Notes
+description: Apache APISIX version 2.10 is released!
+tags: [Release]
+---
+
+> Apache APISIX version 2.10 is released!
+
+<!--truncate-->
+
+Apache APISIX version 2.10 is officially released! 🎉 This is the first LTS release of Apache APISIX with support for 10+ new features and new plugins. Have a quick read to learn about the new features in version 2.10!
+
+## Milestone: The First LTS Telease
+
+This 2.10.0 release is a milestone for Apache APISIX, as Apache APISIX 2.10.0 is our first LTS (Long Time Support) release.
```

Comment:
Would you like to add a download or readme link of APISIX 2.10.0?

Reply:
links are provided in "Download" section in the end of this document.